ui/admin/app/components/form/auth-method/ldap/index.hbs

```diff
@@ -0,0 +1,662 @@
+        <F.Error as |E|>
+          {{#each @model.errors.client_certificate as |error|}}
+            <E.Message>{{error.message}}</E.Message>
+          {{/each}}
+        </F.Error>
+      {{/if}}
+    </Hds::Form::TextInput::Field>
+
+    <Hds::Form::TextInput::Field
+      @isOptional={{true}}
+      @value={{@model.client_certificate_key}}
+      @isInvalid={{@model.errors.client_certificate_key}}
+      @type='password'
+      name='client_certificate_key'
+      disabled={{form.disabled}}
+      {{on 'input' (set-from-event @model 'client_certificate_key')}}
+      as |F|
+    >
+      <F.Label>{{t
+          'resources.auth-method.form.client_certificate_key.label'
+        }}</F.Label>
+      <F.HelperText>{{t
+          'resources.auth-method.form.client_certificate_key.help'
+        }}</F.HelperText>
+      {{#if @model.errors.client_certificate_key}}
+        <F.Error as |E|>
+          {{#each @model.errors.client_certificate_key as |error|}}
+            <E.Message>{{error.message}}</E.Message>
+          {{/each}}
+        </F.Error>
+      {{/if}}
+    </Hds::Form::TextInput::Field>
+  {{/if}}
+
+  <Hds::Form::Toggle::Field
+    name='start_tls'
+    checked={{@model.start_tls}}
+    disabled={{form.disabled}}
+    {{on 'change' this.toggleField}}
+    as |F|
+  >
+    <F.Label>{{t 'resources.auth-method.form.start_tls.label'}}</F.Label>
+    <F.HelperText>{{t
+        'resources.auth-method.form.start_tls.help'
+      }}</F.HelperText>
+  </Hds::Form::Toggle::Field>
+
+  <Hds::Form::Toggle::Field
+    name='insecure_tls'
+    checked={{@model.insecure_tls}}
+    disabled={{form.disabled}}
+    {{on 'change' this.toggleField}}
+    as |F|
+  >
+    <F.Label>{{t 'resources.auth-method.form.insecure_tls.label'}}</F.Label>
+    <F.HelperText>{{t
+        'resources.auth-method.form.insecure_tls.help'
+      }}</F.HelperText>
+  </Hds::Form::Toggle::Field>
+
+  <div class='form-section'>
+    <p class='hds-typography-display-300 hds-font-weight-bold'>{{t
+        'resources.auth-method.section.authenticated-search.title'
+      }}</p>
+    <Hds::Form::HelperText @controlId='helper-text-for-section'>{{t
+        'resources.auth-method.section.authenticated-search.help'
+      }}</Hds::Form::HelperText>
+  </div>
+
+  <Hds::Form::TextInput::Field
+    @isOptional={{true}}
+    @value={{@model.bind_dn}}
+    @isInvalid={{@model.errors.bind_dn}}
+    @type='text'
+    name='bind_dn'
+    disabled={{form.disabled}}
+    {{on 'input' (set-from-event @model 'bind_dn')}}
+    as |F|
+  >
+    <F.Label>{{t 'resources.auth-method.form.bind_dn.label'}}</F.Label>
+    <F.HelperText>{{t 'resources.auth-method.form.bind_dn.help'}}</F.HelperText>
+    {{#if @model.errors.bind_dn}}
+      <F.Error as |E|>
+        {{#each @model.errors.bind_dn as |error|}}
+          <E.Message>{{error.message}}</E.Message>
+        {{/each}}
+      </F.Error>
+    {{/if}}
+  </Hds::Form::TextInput::Field>
+
+  {{#if @model.isNew}}
```

Comment:
The `bind_dn` can be updated but the password can't? Is that correct?

Reply:
The password can be updated, after creating the `bind_password` field returns an hmac so when the model is not new it will use the [secret-input component](https://hashicorp.atlassian.net/browse/ICU-10077). Apologies, i should've added a comment to clarify this.